responsive-web-design-piano/css/style11-1.css

```diff
@@ -0,0 +1,33 @@
+/* box model reset */
+html {
+    box-sizing: border-box;
+}
+
+
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+
+/* piano styles */
+#piano {
+    width: 992px;
+    height: 290px;
+    margin: 80px auto;
+    padding: 90px 20px 0 20px;
+    background-color: #00471b;
+}
+
+
+.keys {
+    width: 949px;
+    height: 180px;
+    padding-left: 2px;
+    background-color: #040404;
+}
+
+/* 
+    It's time set the style for the `.key` class selector.
```

Comment:
It's time to style the keys themselves.
Create a rule that selects the `key` class and
set its `position` to `absolute`.